packages/@aws-cdk/cdk/lib/lazy.ts

```diff
@@ -0,0 +1,166 @@
+import { IResolveContext, Token } from "./token";
+
+/**
+ * Interface for lazy string producers
+ */
+export interface IStringValue {
```

Comment:
Is this not an `IStringValueProducer`? The same applies too... the other `I*Value` interfaces here.